spec/danger_plugin_spec.rb

```diff
@@ -32,7 +33,7 @@ module Danger
 
         it 'accept files as arguments' do
           expect(Swiftlint).to receive(:lint)
-            .with(hash_including(:path => 'spec/fixtures/SwiftFile.swift'))
+            .with(hash_including(:path => File.expand_path('spec/fixtures/SwiftFile.swift')))
```

Comment:
Because of how `find_swift_files` has changed now the returned values are all expanded paths, and so I had to update the spec to take that into account.

I think this is an acceptable price to pay to avoid repeating `File.expand_path` many unnecessary times.